crud/ParserConfigurationController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * @author Patryk Radziszewski <pradziszewski@netis.pl>
+ * @link http://netis.pl/
+ * @copyright Copyright (c) 2015 Netis Sp. z o. o.
+ */
+
+namespace nineinchnick\sync\crud;
+
+use \yii;
+
+class ParserConfigurationController extends \netis\utils\crud\ActiveController
```

Comment:
You can add actions via `actionsClassMap` property of `\netis\utils\crud\ActiveController` when you register this controller in the module so this whole class is not necessary.